### PR TITLE
Add last metric for total registrations

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@rudigiesler ready for review

File: registrations/tests.py [hunk 1]
```diff
@@ -1972,11 +1973,15 @@ def test_created_metric(self):
...
-        [request] = adapter.requests
+        [request1, request2] = adapter.requests
         self._check_request(
-            request, 'POST',
+            request1, 'POST',
             data={"registrations.created.sum": 1.0}
         )
+        self._check_request(
+            request2, 'POST',
+            data={"registrations.created.last": 1}
```
Comment (rudigiesler) [inline registrations/tests.py:1984]: We should probably also check that if we have multiple registrations, that this has a value of 2 for the second time it gets fired, to check that it's actually incrementing properly.

**Review by rudigiesler — COMMENTED** [review]
Looks good, just one comment about an extra test.

**Comment by KaitCrawford** [comment]
@rudigiesler ready for re-review

**Review by rudigiesler — APPROVED** [review]
:+1:

**Pull request merged**